src/resources/Projects/Project.ts

```diff
@@ -57,11 +57,23 @@ export default class Project extends Resource {
         return this.api.delete(this.buildPath(`${Project.baseUrl}/${projectId}`));
     }
 
+    /**
+     * Returns resource IDs grouped by resource status for a given project.
+     *
+     * @param {string} projectId
+     * @returns {Promise<Record<ResourceStatus, string[]>>} A series of resource IDs grouped by their status and resource type.
+     */
+    listResourcesByStatus(projectId: string): Promise<Record<ResourceStatus, string[]>> {
+        return this.api.get<Record<ResourceStatus, string[]>>(
```

Comment:
I think the type here (looking at the example in the swagger doc)

![image](https://github.com/coveo/platform-client/assets/35579930/f79840e2-af93-44b9-9257-e13982702d6a)

Shouldn't it be something like this?
```suggestion
        return this.api.get<Record<ResourceStatus, Record<string, string[]>>>(
```

Reply:
You're absolutely right. Correcting now